Datacard/makeDatacardGGTT_new.py

```diff
@@ -101,18 +101,18 @@ def addMigrationSystematics(df):
   for i in range(nCats-1):    
     template_copy = copy.deepcopy(template)
     template_copy["name"] = template_copy["title"] = "Interpolation_migration_%d_%d"%(i, i+1)
-    systematics_ggtt.experimental_systematics.append(template_copy)
+    systematics_ggbbres.experimental_systematics.append(template_copy)
 
 def grabSystematics(df, args):
   sig_systematics = attemptLoadSystematics(args.sig_syst)
   res_bkg_systematics = attemptLoadSystematics(args.res_bkg_syst)
 
-  addMigrationSystematics(df)
+  #addMigrationSystematics(df)
 
   years = df.year.unique()
 
-  for syst in systematics_ggtt.experimental_systematics:
-    if syst["correlateAcrossYears"] == -1:
+  for syst in systematics_ggbbres.experimental_systematics:
+    if abs(syst["correlateAcrossYears"]) == 1:
```

Comment:
Are you sure about this change? I see in some comments that ```1``` would be full correlation and ```-1``` would be partially correlated

Reply:
To be honest, no. But here is my thinking and let me know if it makes sense:
Previously, the code read
```if syst["correlateAcrossYears"] == -1```
Let's assume that `correlateAcrossYears==1`, which means that we want to correlate the years, i.e. have a common nuisance parameter for all years. In this case, however, this code block would be skipped and we would go to the `else` case which reads
```
else:
  for year in years:
    df["%s_%s"%(syst["name"], year)] = '-'
  df_name = lambda sys_name, year: "%s_%s"%(sys_name, year)
```
which doesn't look like something we should be doing to a fully correlated NP (in reality treating it the same way as a fully uncorrelated one).

That said, it's been a while since I did all of this, and I remember being doubtful about it, so please cross-check.

Reply:
A reminder to revert this bit from our conversation in person

Reply:
Sorry for the delay, This is now reverted, and I believe the PR is ready for merging, @ireed3597.